specs/swagger.yml

```diff
@@ -442,47 +441,77 @@ paths:
           required: true
           type: string
       responses:
-        200:
+        '200':
           description: Success
           schema:
-            $ref: "#/definitions/RegistriesListResponse"
+            $ref: '#/definitions/RegistriesListResponse'
           headers:
             X-Timestamp:
               type: number
               description: FogController server timestamp
-        401:
+        '401':
           description: Not Authorized
-        404: 
+        '404':
+          description: Invalid Node Id
+  '/iofog/{id}/agent/proxy/config':
```

Comment:
GET `/iofog/{id}/agent/proxy` or `/iofog/{id}/agent/proxy-config` would be better imo.